src/main/java/picard/illumina/IlluminaBasecallsToFastq.java

```diff
@@ -201,18 +199,6 @@
     @Option(doc = "Use the new converter", optional = true)
     public boolean USE_NEW_CONVERTER = false;
 
-    @Option(doc = "Maximum mismatches for a barcode to be considered a match.")
-    public int MAX_MISMATCHES = 1;
-
-    @Option(doc = "Minimum difference between number of mismatches in the best and second best barcodes for a barcode to be considered a match.")
-    public int MIN_MISMATCH_DELTA = 1;
-
-    @Option(doc = "Maximum allowable number of no-calls in a barcode read before it is considered unmatchable.")
-    public int MAX_NO_CALLS = 2;
-
-    @Option(doc = "Per-barcode and per-lane metrics written to this file.", shortName = StandardOptionDefinitions.METRICS_FILE_SHORT_NAME, optional = true)
```

Comment:
Basecalls to fastq only created metrics for one release while we were testing and there are lots of guts inside that aren't necessary now that extract barcodes can do all of this.. I'd be hesitant for it to stick around at all and i doubt anyone has used it in that time (1 week or so)